client/src/theme/styles/components/backend/project-collection/_collection-list.scss

```diff
@@ -85,10 +87,8 @@
     flex-grow: 1;
     padding: 3px 0;
     margin-right: 6px;
-    overflow: hidden;
     text-align: left;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    @include textTruncate;
```

Comment:
CIC convention is to add the mixin before other CSS properties in a selector. This is so that you can override the mixin with other declarations if need be (ideally not, but it happens), rather than the mixin overriding your declarations.

Reply:
Sounds good!